Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -151,10 +157,22 @@ open class MessageContentCell: MessageCollectionViewCell {
         let topMessageLabelText = dataSource.messageTopLabelAttributedText(for: message, at: indexPath)
         let bottomMessageLabelText = dataSource.messageBottomLabelAttributedText(for: message, at: indexPath)
 
+        let sentDate = message.sentDate
+        let sentDateString = MessageKitDateFormatter.shared.string(from: sentDate)
+        let timeLabelFont : UIFont = .boldSystemFont(ofSize: 10)
+        let timeLabelColor: UIColor = .darkGray
+        let timeLabelText =
+        NSAttributedString(string: sentDateString, attributes: [NSAttributedString.Key.font: timeLabelFont, NSAttributedString.Key.foregroundColor: timeLabelColor])
+
         cellTopLabel.attributedText = topCellLabelText
         cellBottomLabel.attributedText = bottomCellLabelText
         messageTopLabel.attributedText = topMessageLabelText
         messageBottomLabel.attributedText = bottomMessageLabelText
+        timeLabel.attributedText = timeLabelText
+
+        let displayTimeBySwipingLeft = messagesCollectionView.clipsToBounds == false
```

Comment:
Not sure if this is the best check to have...

Reply:
I tried to make it better.